src/helpers/coordinates.ts

```diff
@@ -28,13 +29,17 @@ export function lettersToNumber(letters: string): number {
   let result = 0;
   const l = letters.length;
   for (let i = 0; i < l; i++) {
-    const charCode = letters.charCodeAt(i);
-    const colIndex = charCode >= 65 && charCode <= 90 ? charCode - 64 : charCode - 96;
+    const colIndex = charToNumber(letters[i]);
     result = result * 26 + colIndex;
   }
   return result - 1;
 }
 
+function charToNumber(char: string) {
+  const charCode = char.charCodeAt(0);
+  return charCode >= 65 && charCode <= 90 ? charCode - 64 : charCode - 96;
```

Comment:
maybe throw if we receiver a non-letter character ?

Reply:
I wouldn't do it. We have typescript for most cases, and this is a hot path were perf matters.